packages/hardware/quirks/devices/AYN Odin 2/bin/battery_led_status

```diff
@@ -0,0 +1,84 @@
+#!/bin/bash
+
+# SPDX-License-Identifier: GPL-2.0-or-later
+# Copyright (C) 2024 ROCKNIX (https://github.com/ROCKNIX)
+
+# Simple script to watch the battery capacity and
+# turn the power LED orange when it reaches 30%, red at 20%, and blinking red at 10%
+
+. /etc/profile
+
+LED_PATH="/sys/class/leds"
+
+function led_brightness() {
+  echo ${2} >${LED_PATH}/${1}/brightness
+}
+
+function led_rgb() {
+  echo ${2} ${3} ${4} >${LED_PATH}/${1}/multi_intensity
```

Comment:
Instead of insisting on using braces here, if you want to make the function a bit safer, you could double-quote the parameters.

Reply:
I'm open to suggestions...